combat-trainer.lic

```diff
@@ -2423,6 +2423,9 @@ class TrainerProcess
     @prayer_mat = settings.prayer_mat
     echo("  @prayer_mat: #{@prayer_mat}") if $debug_mode_ct
 
+    @prayer_mat_storage = settings.prayer_mat_storage || settings.theurgy_supply_container
```

Comment:
`prayer_mat_storage` needs to be in `base.yaml`

Reply:
Fixed!  And updated everywhere to be prayer_mat_container to match the theurgy_supply_container.